pandas/core/indexes/datetimelike.py

```diff
@@ -115,7 +162,7 @@ def _round(self, freq, rounder):
     def round(self, freq, *args, **kwargs):
         return self._round(freq, np.round)
 
-    @Appender(_round_doc % "floor")
+    @Appender(_floor_doc)
```

Comment:
Since you're no longer using a shared docstring, don't use the `Appender`. Instead add the docstring to the method directly.